palantirnet/st-bernards
# MSSB-42: tab order for Provider view

## JIRA Ticket(s)

- See ticket https://palantir.atlassian.net/browse/MSSB-42

## Description

- With client's approval, the radio buttons have been changed to dropdown
<img width="1433" alt="Screen Shot 2020-01-22 at 4 39 06 PM" src="https://user-images.githubusercontent.com/10488517/72941988-f7e99f00-3d37-11ea-80dc-b3b0083c1d08.png">
- With client's request, the second option on the list still exists but is not visible via CSS
<img width="797" alt="Screen Shot 2020-01-22 at 4 39 12 PM" src="https://user-images.githubusercontent.com/10488517/72942022-08017e80-3d38-11ea-93e3-9d8f8f328aff.png">
Mobile:
<img width="321" alt="Screen Shot 2020-01-22 at 4 46 45 PM" src="https://user-images.githubusercontent.com/10488517/72942030-0f288c80-3d38-11ea-9be3-2c768f6ccde1.png">

## Testing instructions
- `drush cim`
- `drush cr`
- `cd web/themes/custom/blrfurther_d8`
- `gulp sass`
- Navigate to the Providers listing page (http://st-bernards.local/providers). Check that the radio buttons are gone and they have been replaced with a dropdown list.
- Tab through the page and make sure all the elements in the exposed filter are accessible.
<img width="1278" alt="Screen Shot 2020-01-22 at 4 58 57 PM" src="https://user-images.githubusercontent.com/10488517/72942244-94ac3c80-3d38-11ea-9075-b0fee4622d91.png">
